### PR TITLE
Rename all of the Jakarta REST artifacts from "jaxrs" to "rest"

### PR DESCRIPTION
**Pull request by Harsh4902**
- Changed all files or folder with name 'jaxrs' to 'rest'.
- Also changed references in .adoc file.
- Fixes issue: #90

### CODE REVIEW

File: src/main/antora/modules/websvcs/pages/rest/rest003.adoc [hunk 1]
```diff
@@ -8,13 +8,13 @@ This section demonstrates the steps that are needed to create, build, deploy, an
 This section explains how to use NetBeans IDE to create a RESTful web service using a Maven archetype.
 The archetype generates a skeleton for the application, and you simply need to implement the appropriate method.
 
-You can find a version of this application at `_jakartaee-examples_/tutorial/jaxrs/hello/`.
+You can find a version of this application at `_jakartaee-examples_/tutorial/rest/hello/`.
```
Comment (kito99) [inline src/main/antora/modules/websvcs/pages/rest/rest003.adoc:11]: The references in here point to the jakartaee-examples repo: https://github.com/eclipse-ee4j/jakartaee-examples. Can you raise a PR for that repo as well to change the folder names?

File: src/main/antora/modules/websvcs/pages/rest/rest003.adoc [hunk 2]
```diff
@@ -8,13 +8,13 @@ This section demonstrates the steps that are needed to create, build, deploy, an
...
 
 ==== To Create a RESTful Web Service Using NetBeans IDE
 
 . Ensure you have installed the tutorial archetypes as described in xref:intro:usingexamples/usingexamples.adoc#_installing_the_tutorial_archetypes[Installing the Tutorial Archetypes].
 
-. In NetBeans IDE, create a simple web application using the `jaxrs-service-archetype` Maven archetype.
+. In NetBeans IDE, create a simple web application using the `rest-service-archetype` Maven archetype.
```
Comment (kito99) [inline src/main/antora/modules/websvcs/pages/rest/rest003.adoc:17]: The archetype name may still start with `jaxrs`. Can you check? If not, I can.

**Review by kito99 — CHANGES_REQUESTED** [review]
Looks good! I just have some questions/comments about the examples.

@omniprof may be recreating these examples, too (that just occurred to me). Ken, can you chime in?

File: src/main/antora/modules/websvcs/pages/rest/rest003.adoc (hunk 1 above)
Comment (Harsh4902) [reply src/main/antora/modules/websvcs/pages/rest/rest003.adoc:11]: Okay, I will raise PR for that.

File: src/main/antora/modules/websvcs/pages/rest/rest003.adoc (hunk 2 above)
Comment (Harsh4902) [reply src/main/antora/modules/websvcs/pages/rest/rest003.adoc:17]: Okay, let me check

Comment (kito99) [reply src/main/antora/modules/websvcs/pages/rest/rest003.adoc]: @Harsh4902 so was this updated?

Comment (Harsh4902) [reply src/main/antora/modules/websvcs/pages/rest/rest003.adoc]: @kito99 Yes, I have changed 'rest' with 'jaxrs'.

**Review by kito99 — APPROVED** [review]

**Pull request merged**